e2e/src/import-storage/index.test.ts

```diff
@@ -2,7 +2,7 @@ import { describe, expect, it } from 'vitest'
 import path from 'path'
 
 import { api, chain, setupApi } from '../helper'
-import { importStorage, overrideWasm } from '../../src/utils/import-storage'
+import { importStorage, overrideWasm } from '@acala-network/chopsticks/lib/utils/import-storage'
```

Comment:
```suggestion
import { importStorage, overrideWasm } from '@acala-network/chopsticks/src/utils/import-storage'
```

It will be best if we run tests without having to build first